docs/current/guides/snippets/use-services/start-stop-services/main.go

```diff
@@ -0,0 +1,50 @@
+package main_test
+
+import (
```

Comment:
@helderco @TomChv @slumbering could you help me with equivalent code in Typescript and Python?

Reply:
I've also pushed the equivalent for that one. However I'm still facing the same error :
![Capture d’écran 2023-10-16 à 14 07 42](https://github.com/dagger/dagger/assets/1186424/f9d8a756-8176-41bb-8b03-a0a17eb7787f)

Any idea @vito ?

Reply:
Is this also resolved by wrapping in `./hack/dev` as above? No clue what that error means. It seems like it's hitting a completely different service than Dagger.